src/EndpointV2/EndpointV2Middleware.php

```diff
@@ -98,8 +99,14 @@ public function __invoke(CommandInterface $command)
         $operation = $this->api->getOperation($command->getName());
         $commandArgs = $command->toArray();
         $providerArgs = $this->resolveArgs($commandArgs, $operation);
+        if (!empty($providerArgs[self::ACCOUNT_ID_PARAM])) {
+            $command->getMetricsBuilder()->append(MetricsBuilder::RESOLVED_ACCOUNT_ID);
+        }
         $endpoint = $this->endpointProvider->resolveEndpoint($providerArgs);
-
+        $command->getMetricsBuilder()->identifyMetricByValueAndAppend(
+            'account_id_endpoint',
+            $endpoint->getUrl()
```

Comment:
question: does the spec really call for the url?

Reply:
Actually this feature was removed from the spec. I will remove it.